.github/workflows/testsprite-ci.yml

```
@@ -0,0 +1,78 @@
+          node dist/index.js prescan --help 2>&1 | head -20
+
+      - name: Run CLI unit tests
+        run: |
+          cd cli
+          bun run test -- --testPathIgnorePatterns 'test/(integration|load)\.' --testTimeout 60000
+        timeout-minutes: 5
+
+  testsprite-e2e:
+    name: TestSprite E2E (if configured)
+    runs-on: ubuntu-latest
+    if: ${{ secrets.TESTSPRITE_API_KEY != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)